src/libcore/intrinsics.rs

```diff
@@ -520,6 +507,35 @@ extern "rust-intrinsic" {
     pub fn bswap32(x: u32) -> u32;
     /// Reverses the bytes in a `u64`.
     pub fn bswap64(x: u64) -> u64;
+}
+/// The following intrinsics are disallowed on PNaCl:
+/// We sneakily redirect them to libm or to a larger sized intrinsic.
```

Comment:
I forgot to mention that the PNaCl pexe format doesn't define most of the LLVM intrinsics and the IR verilator will throw out modules that try to use them.